Dashboard/app/js/lib/views/views.js

```diff
@@ -93,12 +93,30 @@ Ember.Handlebars.registerHelper('tooltip', function (i18nKey) {
 
 
 Ember.Handlebars.registerHelper('placemarkDetail', function () {
-  var answer, markup, question;
+  var answer, markup, question, cascadeJson, cascadeString = "";
 
   question = Ember.get(this, 'questionText');
   answer = Ember.get(this, 'stringValue').replace(/\|/g, ' | '); // geo data
   answer = answer.replace(/\//g, ' / '); // also split folder paths
 
+  if (Ember.get(this, 'questionType') === 'CASCADE') {
+
+      if (answer.indexOf("|") > -1) {
+        // ignore
+      } else {
+        cascadeJson = JSON.parse(answer);
+        for (var i = 0; i < cascadeJson.length; i++) {
```

Comment:
Could we use [Array.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) here as well?